docs/.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Thanks a bunch for adding this! I was running into issues with the previous `eslintrc` conflicting with what I had in the root repo, since we use `@typescript/eslint` and need some additional parser config stuff.